generate_stats.py

```diff
@@ -51,11 +63,19 @@ def collect_statistics(my_core, root_set):
  
     """
     stats = {}
+    data = {}
+    
     dagmc_tags = dagmc_stats.get_dagmc_tags(my_core)
+    
     entity_types = [types.MBVERTEX, types.MBTRI, types.MBENTITYSET]
-    stats['Native Ranges'] = dagmc_stats.get_native_ranges(my_core, root_set, entity_types)     # get Ranges of various entities
-    stats['EntitySet Ranges'] = dagmc_stats.get_entityset_ranges(my_core, root_set, dagmc_tags['geom_dim']) 
-    return stats
+    native_ranges = dagmc_stats.get_native_ranges(my_core, root_set, entity_types)     # get Ranges of various entities
+    
+    entityset_ranges = dagmc_stats.get_entityset_ranges(my_core, root_set, dagmc_tags['geom_dim'])
+    
+    data['S_P_V_data'] = dagmc_stats.get_surfaces_per_volume(my_core, entityset_ranges)
```

Comment:
Good idea to separate this into two dictionaries!  It might make life easier down the road if you use the same key for both, possibly even using a variable to store that key for consistency:

```
s_p_v_key = ‘S_P_V’
data[s_p_v_key] = ....
stats[s_p_v_key] = ...
```